packages/ios-poc/src/launchIOS.ts

```diff
@@ -84,7 +84,9 @@ const launchTest = async ({
     stdio: "inherit",
   });
   const recordingPromise = startRecord(simulatorId, traceFile);
-
+  execSync(`sleep 2`, {
```

Comment:
could you add a command on why we need to sleep here?

Reply:
Refacto